node/src/components/block_validator/tests.rs

```diff
@@ -224,594 +245,351 @@ pub(super) fn new_transfer(rng: &mut TestRng, timestamp: Timestamp, ttl: TimeDif
     let ttl = TimeDiff::from_millis(200);
     let timestamp = Timestamp::from(1000);
-    let deploy1 = new_deploy(&mut rng, timestamp, ttl);
-    let deploy2 = new_deploy(&mut rng, timestamp, ttl);
-    let transfer1 = new_transfer(&mut rng, timestamp, ttl);
-    let transfer2 = new_transfer(&mut rng, timestamp, ttl);
+    let deploy_legacy = Transaction::from(new_legacy_deploy(&mut rng, timestamp, ttl));
+    let transaction_v1 = Transaction::from(new_v1_standard(&mut rng, timestamp, ttl));
+    let transfer_legacy = Transaction::from(new_legacy_transfer(&mut rng, timestamp, ttl));
+    let transfer_v1 = Transaction::from(new_v1_transfer(&mut rng, timestamp, ttl));
 
     // First we make sure that our transfers and deploys would normally be valid.
-    let deploys = vec![deploy1.clone(), deploy2.clone()];
-    let transfers = vec![transfer1.clone(), transfer2.clone()];
-    let mut context = ValidationContext::new()
-        .with_num_validators(&mut rng, 1)
-        .with_deploys(deploys)
-        .with_transfers(transfers)
-        .include_all_deploys()
-        .include_all_transfers();
-    assert!(context.validate_block(&mut rng, timestamp).await);
-
-    // Now we hide a transfer in the deploys section. This should be invalid.
-    let deploys = vec![deploy1.clone(), deploy2.clone(), transfer2.clone()];
-    let transfers = vec![transfer1.clone()];
-    let mut context = ValidationContext::new()
-        .with_num_validators(&mut rng, 1)
-        .with_deploys(deploys)
-        .with_transfers(transfers)
-        .include_all_deploys()
-        .include_all_transfers();
-    assert!(!context.validate_block(&mut rng, timestamp).await);
-
-    // A regular deploy in the transfers section is also invalid.
-    let deploys = vec![deploy2.clone()];
-    let transfers = vec![transfer1.clone(), deploy1.clone(), transfer2.clone()];
-    let mut context = ValidationContext::new()
-        .with_num_validators(&mut rng, 1)
-        .with_deploys(deploys)
-        .with_transfers(transfers)
-        .include_all_deploys()
-        .include_all_transfers();
-    assert!(!context.validate_block(&mut rng, timestamp).await);
-
-    // Each deploy must be unique
-    let deploys = vec![deploy1.clone(), deploy2.clone()];
-    let transfers = vec![transfer1.clone(), transfer2.clone()];
-    let mut context = ValidationContext::new()
-        .with_num_validators(&mut rng, 1)
-        .with_deploys(deploys)
-        .with_transfers(transfers)
-        .include_all_deploys()
-        .include_all_transfers()
-        .include_deploys(vec![TransactionHashWithApprovals::from(
-            &Transaction::Deploy(deploy1.clone()),
-        )]);
-    assert!(!context.validate_block(&mut rng, timestamp).await);
+    let transactions = vec![deploy_legacy.clone(), transaction_v1.clone()];
+    let transfers = vec![transfer_legacy.clone(), transfer_v1.clone()];
+    assert!(validate_block(&mut rng, timestamp, transactions, transfers, vec![], vec![]).await);
```

Comment:
This reverts the changes related to `ValidationContext` introduced by @fizyk20 in [this PR](https://github.com/casper-network/casper-node/pull/4543). Is this intended?

Reply:
im not spun up on the details of that...almost certainly unintentional merge clobbering. @darthsiroftardis can you look into this?

Reply:
Yes!